src/replicache.ts

```diff
@@ -264,6 +260,16 @@ export default class Replicache implements ReadTransaction {
     }
   }
 
+  private _schedulePush(interval: number | null): void {
+    // We do not want to restart the push timer.
```

Comment:
Let's clarify this comment, since this is an important design decision:

```
// To make multiuser collab feel as live as possible, we need events to be sent very soon
// after they are generated. Even if a new event comes in, we do *not* delay the initial one
// from being sent. Any new events that gets in before the timer elapses will get sent too,
// but we do not delay the train from leaving.
```